client-src/default/index.js

```diff
@@ -2,7 +2,7 @@
 
 /* global __resourceQuery WorkerGlobalScope self */
 /* eslint prefer-destructuring: off */
-
+const querystring = require('querystring');
```

Comment:
```suggestion
const qs = require('querystring');
```

Reply:
Disagree, no need rewrite full package name to short name.

Reply:
`qs` also conflicts with another variable in the package. That's why it's been renamed from the original PR.

Reply:
@trescenzi you did everything right, do not rename